docs/getting-started/configure-via-openshift-ui.md

```diff
@@ -0,0 +1,59 @@
+## Configuring VDO via Openshift Web Console
+
+The easiest way to configure VDO is to configure the CRD's via Openshift Web Console, follow the steps below for the configuration.
+
+#### Pre-requisite
+
+You should have below data in handy.
+
+- IP address of vcenter
+- Secure Connection - If you choose to establish a secure connection to vcenter, you need to provide a ssl thumbprint
+- Login credentials for vcenter
+- Datacenter(s) - you can provide a comma separated list of datacenters
+
+
+### Step-1
+Make sure the VDO is up and running, for this you can go to `Installed Operator` side menu and confirm the status of the operator as Succeeded.
+
+### Step-2
+Create the secret with the credentials for your VC of a type `Source secret`
```

Comment:
Now that we have installed VDO, we need to setup few pre-requisite to configure VDO.
Lets create a secret to store the credentials with which you would want CSI and CPI to connect to VC. Please note if CPI and CSI use different credentials then create secret for each set of credentials. This will be required to configure cloud and storage providers